app/scripts/modules/netflix/fastProperties/view/fastPropertyDetails.controller.ts

```diff
@@ -0,0 +1,90 @@
+import {module} from 'angular';
+
+import { UPDATE_FAST_PROPERTY_WIZARD_CONTROLLER } from '../wizard/updateFastPropertyWizard.controller';
+import { DELETE_FAST_PROPERTY_WIZARD_CONTROLLER } from '../wizard/deleteFastPropertyWizard.controller';
+import { FAST_PROPERTY_READER_SERVICE } from '../fastProperty.read.service';
+import {Application} from '../../../core/application/application.model';
+import {Property} from '../domain/property.domain';
+
+
+export class FastPropertyDetailsController {
+
+  public property: Property;
+
+  static get $inject() { return [
+    '$uibModal',
+    'fastPropertyReader',
+    'fastProperty',
+    'app'
+  ];
+  }
+
+  constructor(
+    private $uibModal: any,
+    private fastPropertyReader: any,
```

Comment:
this can be `FastPropertyReaderService` (which I would of course rename to `FastPropertyReadService` or `FastPropertyReader`, but that is just me) instead of `any`